config.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"errors"
+	"os"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+// ConfigsModel ...
+type ConfigsModel struct {
+	CloneIntoDir  string
+	RepositoryURL string
+	Commit        string
+	Tag           string
+	Branch        string
+	CloneDepth    string
+
+	PullRequestURI         string
```

Comment:
I'd rename this to `PullRequestRepositoryURL` or rather `PullRequestRepositoryCloneURL`, as `PullRequestURI` sounds like the URL of the PR page (something like e.g. https://github.com/bitrise-io/steps-git-clone/pull/52)